src/main/java/org/jenkinsci/plugins/configfiles/buildwrapper/ConfigFileBuildWrapper.java

```diff
@@ -102,42 +100,23 @@ public boolean isApplicable(AbstractProject<?, ?> item) {
 
     }
 
-    /**
-     * fix for JENKINS-17555
-     */
-    public class ManagedFilesEnvironment extends Environment {
```

Comment:
Up to plugin maintainer I guess. [Search](https://github.com/search?utf8=%E2%9C%93&q=user%3Ajenkinsci+ManagedFilesEnvironment&type=Code&ref=searchresults) turns up no outside uses in public plugins; seems highly unlikely there would be one (no apparent reason for this ever to have been `public`).